.github/actions/setup-integration-test/action.yml

```diff
@@ -1,12 +1,12 @@
 name: Setup Integration Test Action
-description: "A composite GitHub action used to download the Chime JS SDK repo, install dependencies and setup SauceLabs Connect."
+description: "A composite GitHub action used to download the Chime JS SDK repo, install dependencies and setup testing environment."
 inputs:
   sauce-username:
-    required: true
+    required: false
```

Comment:
So this runs in headless mode in GitHub environment? Does this change impact any other runs? Guess not.

Reply:
The other runs no longer work :)

Reply:
Why dont we need this anymore?